### PR TITLE
fix(HIG-3688): popover for sessions and errors

### PR DESCRIPTION
**Pull request by aptlin**
## Summary

Screenshot 2023-01-23 at 2 40 29 PM

## How did you test this change?

local click test

## Are there any deployment considerations?

no

### CODE REVIEW

File: frontend/src/components/KeyboardShortcutsEducation/KeyboardShortcutsEducation.tsx [hunk 1]
```diff
@@ -377,11 +377,11 @@ export const ErrorsKeyboardShortcuts: ShortcutItem[] = [
 	},
 	{
 		description: `Next error`,
-		shortcut: ['j'],
+		shortcut: ['k'],
 	},
 	{
 		description: `Previous error`,
-		shortcut: ['k'],
+		shortcut: ['j'],
```
Comment (ccschmitz) [inline frontend/src/components/KeyboardShortcutsEducation/KeyboardShortcutsEducation.tsx:384]: What's the reason for swapping these? `j` for moving down a list and `k` for moving up are pretty standardized. I try them in almost every app I use that presents a list of items I can interact with and there's a lot of muscle memory built up from apps like Gmail, GitHub, Linear, etc. This makes them behave the opposite of what I would expect.

**Review by ccschmitz — CHANGES_REQUESTED** [review]
Code looks great, but I'm wondering about the behavior swap for `j` and `k` - it's the opposite of all the muscle memory I've developed using these shortcuts in other apps. Would you mind sharing more about why we're changing it?

File: frontend/src/components/KeyboardShortcutsEducation/KeyboardShortcutsEducation.tsx (hunk 1 above)
Comment (aptlin) [reply frontend/src/components/KeyboardShortcutsEducation/KeyboardShortcutsEducation.tsx:384]: Thanks for catching this! Fixed in [4438ce2](https://github.com/highlight/highlight/pull/3578/commits/4438ce23f4c83dd42bbdcc81ccdb15ded550d9fd)

Comment (ccschmitz) [reply frontend/src/components/KeyboardShortcutsEducation/KeyboardShortcutsEducation.tsx:384]: Nice! Do we still need to revert the changes to these keyboard shortcut docs?

**Comment by aptlin** [comment]
Got confused by the vim-like `down j - up k` vs our `up - down` buttons that do not match with the QWERTY / Dvorak ordering of j and k. 
Screenshot 2023-01-26 at 9 35 51 AM
Fixed now

**Review by ccschmitz — APPROVED** [review]

**Pull request merged**